runtime/common/NoiseModel.h

```diff
@@ -297,4 +298,38 @@ class phase_flip_channel : public kraus_channel {
     validateCompleteness();
   }
 };
+
+/// @brief readout_error is a type of error that Models
+/// the probabilistic errors that can happen when measuring qubits.
+/// It allows to define a custom probability distribution that describes how
+/// often a qubit's measurement result differs from its actual state.
+/// Its constructor expects a 2x2 matrix of float values that represents the
+/// probability matrix applied to the probabilistic state of the qubit measurement.
+/// It takes the form of
+/// P = [[p(0|0), p(0|1)], [p(1|0), p(1,1)]]
+/// where p(i|j) is the probability of measuring outcome i when the actual state is j
+class readout_error : public kraus_channel {
```

Comment:
Should we use camel case or at least make class names start with capital letters? I wouldn't have known it was a class unless I saw the implementation.

Reply:
Cudaq for whatever reason uses snake_case for noise mode API